src/main/java/DayThree.java

```diff
@@ -22,9 +25,11 @@
                 resultList.set(position, numberToAdd);
             }
             else{
+                //ever false?
                 if(resultList.get(numberToCheck).equals("2"))
                 {
                     int position = Integer.valueOf(resultList.get(numberToCheck+3));
+                    //naming
```

Comment:
Because of the duplication, we have some really misleading naming here, everything is still called `add` when in this section we're dealing with multiplication.